src/engraving/libmscore/beam.cpp

```diff
@@ -1001,11 +1006,9 @@ void Beam::layout2(std::vector<ChordRest*> chordRests, SpannerSegmentType, int f
         LayoutBeams::respace(&chordRests);
     }
 
-    // FIXME
-    if (!chordRests.front()->isChord() || !chordRests.back()->isChord()) {
-        NOT_IMPL_RETURN;
-    }
+    _beamSpacing = score()->styleB(Sid::useWideBeams) ? 4 : 3;
 
+    // todo: add edge case for when a beam starts or ends on a rest
```

Comment:
The problem is that `toChord` will cause a crash (assertion failure) when the item is not a chord. The VTests contain that scenario, which is why I added a check for that.

Reply:
Oh, yes. I removed that when testing because it was causing another issue with this specific PR and forgot to add that back in.